Changelog.md

```diff
@@ -8,6 +8,7 @@ Compiler Features:
  * Commandline Interface: Re-group help screen.
  * Output compilation error codes in standard-json and when using ``--error-codes``.
  * Yul: Raise warning for switch statements that only have a default and no other cases.
+ * NatSpec: Add fields "kind" and "version" to the JSON output.
```

Comment:
Wrong changelog.

Reply:
What about it is wrong? That it is missing "documentation"? I figured the "NatSpec:" part would clarify that.

Reply:
@Marenz We already released 0.6.10. This should go in the unreleased section. I didn't realize this either ;)

Reply:
ooh, will fix, thank you